src/logmessage.cpp

```diff
@@ -123,10 +123,11 @@ namespace g3 {
    }
 
 
-   LogMessage::LogMessage(const std::string& file, const int line,
+   LogMessage::LogMessage(const char * file, const int line,
```

Comment:
then I think the std::string parameter can be left too as it was originally, right?